Plugin/OrderPlugin.php

```diff
@@ -113,6 +121,10 @@ public function beforeSetStatus(Order $subject, $status)
             return [Order::STATE_PROCESSING];
         }
         
+        if ($this->featureSwitches->isDisallowOrderStatusOverride()) {
```

Comment:
How about others methods in this file? Do I understand correctly that with this feature switch those methods do not change anything?
If so I suggest two options to improve readability:
- check feature switch and early return in each method
or
- add comment / comments

Reply:
Recharge customer with an existing Bolt credit card is a custom behavior of Bolt plugin,  so feature switch M2_DISALLOW_ORDER_STATUS_OVERRIDE does not affect it. 

Then I have added comments for changes.